graylog2-web-interface/src/views/logic/views/GetPositionForNewWidget.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import { widgetDefinition } from 'views/logic/Widgets';
+import WidgetPosition from 'views/logic/widgets/WidgetPosition';
+import Widget from 'views/logic/widgets/Widget';
+import View from 'views/logic/views/View';
+
+const GetPositionForNewWidget = (widget: Widget, queryId: string, view: View) => {
+  const widgetDef = widgetDefinition(widget.type);
```

Comment:
This logic is contained in `AddNewWidgetsToPositions` as well. Can we consolidate it?

Reply:
I do not really see how it would it improve the code. :confused: